mesonbuild/compilers/cpp.py

```diff
@@ -824,8 +824,9 @@ def get_compiler_check_args(self, mode: CompileCheckMode) -> T.List[str]:
         return []
 
 class C2000CPPCompiler(C2000Compiler, CPPCompiler):
-    def __init__(self, exelist: T.List[str], version: str, for_machine: MachineChoice, is_cross: bool,
-                 info: 'MachineInfo', exe_wrapper: T.Optional['ExternalProgram'] = None,
+    def __init__(self, exelist: T.List[str], version: str, for_machine: MachineChoice,
+                 is_cross: bool, info: 'MachineInfo',
+                 exe_wrapper: T.Optional['ExternalProgram'] = None,
```

Comment:
Why is this just being reflowed without changes?

Reply:
Good point, both the inherited `__init__` methods will be called anyway won't they?

Reply:
I think it is basically to set exe_wrapper, linker and full_version to None?

Reply:
Without this I end up with `TypeError: __init__() got an unexpected keyword argument 'full_version'`

Reply:
The `__init__` method belongs there.

My question is why is this formatting change included in the PR, you are not adding any code changes to this init method?

Reply:
Ah I see, I think I was comparing with the `class C2000CCompiler(C2000Compiler, CCompiler):` which had it formatted slightly differently. I will revert the whitespace change.

Reply:
Thanks! The diff is much clearer now.